src/System.Security.Cryptography.X509Certificates/tests/PublicKeyTests.cs

```diff
@@ -125,5 +126,49 @@ public static void TestKey_RSA()
                 Assert.Equal(expectedExponent, rsaParameters.Exponent);
             }
         }
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void TestKey_ECDsaCng256()
+        {
+            TestKey_ECDsaCng(TestData.ECDsa256Certificate, TestData.ECDsaCng256PublicKey);
+        }
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void TestKey_ECDsaCng384()
+        {
+            TestKey_ECDsaCng(TestData.ECDsa384Certificate, TestData.ECDsaCng384PublicKey);
+        }
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void TestKey_ECDsaCng521()
+        {
+            TestKey_ECDsaCng(TestData.ECDsa521Certificate, TestData.ECDsaCng521PublicKey);
+        }
+
+        private static void TestKey_ECDsaCng(byte[] certBytes, TestData.ECDsaCngKeyValues expected)
+        {
+#if !NETNATIVE
```

Comment:
I actually don't know if this will make the test system happy or not.  If not, we'll find out eventually :).